include/freerdp/input.h

```diff
@@ -97,7 +97,10 @@ extern "C"
 		pRelMouseEvent RelMouseEvent;               /* 23 */
 		pQoEEvent QoEEvent;                         /* 24 */
 
-		UINT32 paddingB[32 - 25]; /* 25 */
+		UINT32 lastInputTimestamp;                  /* 25 */
+		UINT16 lastX; UINT16 lastY;                 /* 26 */
```

Comment:
wrong numbering, this is public API.
also (since you only need to access these in `input.c`, better move to `rdp_input_internal` and hide it. (sorry, missed that)